lib/action_view/component/am_monkey_patch.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
```

Comment:
How about we call this file `active_model_conversion_monkey_patch`? I'd prefer to avoid abbreviations/shorthand.